stdlib/public/runtime/CygwinPort.cpp

```diff
@@ -0,0 +1,131 @@
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <windows.h>
+#include <psapi.h>
+#include <mutex>
+
+struct dl_phdr_info
+{
+	void			 *dlpi_addr;
+	const char       *dlpi_name;
+};
+
+int dl_iterate_phdr(int(*callback)(struct dl_phdr_info *info, size_t size, void *data),
```

Comment:
Please don't export symbols with names other than `swift_*` from the runtime.  I understand that you need a shim layer to make existing code work with WinAPI.  Let's create one without creating extra exports.